src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/MessageBoxInstance.cs

```diff
@@ -52,11 +52,21 @@ public class MessageBoxInstance
         /// </summary>
         public DateTime? DueDateTime { get; set; }
 
+        /// <summary>
+        /// Delete status of instance; default or softDeleted.
+        /// </summary>
```

Comment:
so the string value of the DeleteStatus must be either "default" or "softdeleted"?

Reply:
yeah, or that's at least the two values currently used in SBL. I guess it could be expanded if we see the need in the future.